### PR TITLE
Added output parameter for the response time of the request in the RESTful API client addon

### PR DESCRIPTION
**Pull request by Artem-Kuz**
Users can now get the time it took for a response to complete as an output parameter. 

Signed-off-by: Artem Kuznetsov <artem.kuznetsov@testproject.io>

### CODE REVIEW

File: restful-api-client/src/main/java/io/testproject/addon/restfulapiclient/actions/base/BaseAction.java [hunk 1]
```diff
@@ -59,6 +59,9 @@
     @Parameter(description = "Response Headers", direction = ParameterDirection.OUTPUT)
     public String responseHeaders;
 
+    @Parameter(description = "Response time of the request", direction = ParameterDirection.OUTPUT)
```
Comment (mstrelex) [inline restful-api-client/src/main/java/io/testproject/addon/restfulapiclient/actions/base/BaseAction.java:62]: @Artem-Kuz can you please add the time units specification in the description, is it I'm millis?

Comment (Artem-Kuz) [reply restful-api-client/src/main/java/io/testproject/addon/restfulapiclient/actions/base/BaseAction.java]: It is in hh:mm:ss format

Comment (mstrelex) [reply restful-api-client/src/main/java/io/testproject/addon/restfulapiclient/actions/base/BaseAction.java]: I don't think that it's good enough, usually, REST requests are measured in milliseconds. Please adjust.

Comment (Artem-Kuz) [reply restful-api-client/src/main/java/io/testproject/addon/restfulapiclient/actions/base/BaseAction.java]: Done

**Review by mstrelex — APPROVED** [review]

**Pull request merged**